app/main.py

```diff
@@ -16,4 +16,7 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {"mutable": [my_favourite_films,
```

Comment:
Please make it look prettier and more readable.
Place each variable from the new line
Something like this:
```
sorted_variables = {
    "mutable": [my_favourite_films, collection_of_coins, marks],
    ...
```

Reply:
> Please make it look prettier and more readable. Place each variable from the new line Something like this:
> 
> ```
> sorted_variables = {
>     "mutable": [my_favourite_films, collection_of_coins, marks],
>     ...
> ```

i fixed it